cpu/esp8266/esp-wifi/esp_wifi_netdev.c

```diff
@@ -455,18 +451,11 @@ static int IRAM _send(netdev_t *netdev, const iolist_t *iolist)
 
     if (res == ERR_OK) {
         /* There was no ieee80211_output_pbuf error and no send timeout. */
-#ifdef MODULE_NETSTATS_L2
-        netdev->stats.tx_bytes += iol_len;
-        netdev->event_callback(netdev, NETDEV_EVENT_TX_COMPLETE);
```

Comment:
this should stay in somehow, see other netdev implementations, e.g. also esp32 esp_wifi

Reply:
Yes, of course you are right, thanks.